py/builtinimport.c

```diff
@@ -334,6 +334,10 @@ mp_obj_t mp_builtin___import__(size_t n_args, const mp_obj_t *args) {
         mod_len = new_mod_l;
     }
 
+    if (mod_len == 0) {
+        mp_raise_ValueError("empty module name");
```

Comment:
Considering that it's very rare to hit this and unlikely in most applications (must be obtained from calling `__import__('')`), and that this code is included in all builds, I think it's fair enough to have a blank error message and just use `mp_raise_ValueError(NULL)`.  This will reduce code size.

Reply:
Indeed! I made the change.